smozROM.py

```diff
@@ -948,8 +948,8 @@ def getBaseWeight(romByteArray):
 
                 if weightBugFix.toApply or ratingScaleOption == 15 or romNewBaseWeight != romOldBaseWeight:
                     print("\n  Before you save, please read this:" + "\n")
-                    print("     If you are using the 0-15 ratings or different year base weight, then you" + "\n"
-                        "     need to update NOSE.  See 'NOSE update.html' for more information." + "\n")
+                    print("     If you are using the 0-15 ratings or different year base weight with NOSE," + "\n"
+                        "     then you need to use version 1.2d or higher.  See 'NOSE update.html' for more information." + "\n")
```

Comment:
Printed line lengths should max out at 79 characters since the Windows CMD prompt doens't auto-wrap. This line is too long now.